Packs/Workday/ReleaseNotes/1_0_6.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### New: Workday IAM Event Generator (Beta)
+- IAM mock Workday full reports and events. (Available from Cortex XSOAR 6.0.0).
```

Comment:
This integration generates mock Workday reports and events, which you should use for testing and development purposes. (Available from Cortex XSOAR 6.0.0).